### PR TITLE
Allow an environment argument to support extension configuration

### PR DESCRIPTION
**Pull request by fxchen**
Context: https://discord.com/channels/1092917289164230770/1093020282891026554/1093936253478633542

Usage:
```
python3 babyagi.py -e .env .env.test # optionally more files here
```

### CODE REVIEW

**Review by francip — COMMENTED** [review]
I was wondering if we can make it load additional .env instead of replacing the main one. Kind of like layering/include. This way we can have ENABLE_EXTENSIONS=<extensions .env> and load additional stuff from there.

**Comment by fxchen** [comment]
@francip - ✅ `ENABLE_EXTENSIONS` can be one of the additional variables in `.env.test`

File: babyagi.py [hunk 1]
```diff
@@ -9,7 +10,18 @@
...
 import os
 
-# Set Variables
+# Parse arguments for optional extensions
+parser = argparse.ArgumentParser()
+parser.add_argument('-e', '--env', nargs='+', help='filenames for env')
+args = parser.parse_args()
+
+# Set environment variables for optional extensions
+if args.env:
```
[the remark this answers is not in the document]
Comment (fxchen) [reply babyagi.py:22]: Yes fair point!

**Review by francip — APPROVED** [review]

**Comment by francip** [comment]
Ugh, not enough sleep. Need to check if this messes up with .env.example and if so, we need to fix this.

**Pull request merged**